Add changelog.json for end user facing changes in KKP 2.22.0

embik opened the pull request:
**What this PR does / why we need it**:

I've extracted what I considered the most important end user (i.e. someone who uses the dashboard of a KKP instance) facing changes and updated the `changelog.json` with it. I assume it does not cover everything, but unless I've missed something crucial it's probably a solid amount of information for users. The UI links to the full changelog anyway as well.

**Which issue(s) this PR fixes**:
<!--optional, in `fixes #<issue number>` format, will close the issue(s) when PR gets merged-->
Fixes #

**What type of PR is this?**
/kind documentation

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change? Then add your Release Note here**:
<!--
Write your release note:
1. Enter your extended release note in the below block. If the PR requires additional action from users switching to the new release, include the string "action required".
2. If no release note is required, just write "NONE".
-->
```release-note
NONE
```

**Documentation**:
<!--
Please do one of the following options:
- Add a link to the existing documentation
- Add a link to the kubermatic/docs pull request
- If no documentation change is applicable then add:
  - TBD (documentation will be added later)
  - NONE (no documentation needed for this PR)
-->
```documentation
NONE
```

embik commented:
/retest

pkprzekwas commented on modules/web/src/assets/config/changelog.json:
```diff
@@ -1,4 +1,185 @@
+        "category": "added",
+        "description": "Add Cilium CNI v1.13.0."
+    },
+
```
Are you sure those empty lines don't affect JSON parsing?

embik replied:
I tried it by running the dashboard locally and it was fine, I was able to see the full changelog via it.

Waseem826 commented:
I think we've missed following features:
- Default Project Quota
- Live estimation of Project Quota while creating cluster/MD.
- vSphere tag categories
- Cluster RBAC changes (maybe)
- Edit/Customize cluster templates
- Default application installation when Cilium is selected as CNI plugin

embik commented:
> I think we've missed following features:
> 
> * Default Project Quota
> * Live estimation of Project Quota while creating cluster/MD.

Added that, but I suspect we should look into having changelog items only showing up for EE installations.

> * vSphere tag categories

I'm not sure what to write here to be honest since I don't fully understand the changes.

> * Cluster RBAC changes (maybe)

Not sure here either, sorry.

> * Edit/Customize cluster templates

Added.

> * Default application installation when Cilium is selected as CNI plugin

I think this is more of an internal change. It's still on the changelog, but I'm not sure it should be on the abridged version.

Waseem826 commented:
/approve
